seleniumtests/selenium_test.py

```diff
@@ -0,0 +1,20 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from webdriver_manager.chrome import ChromeDriverManager
+import unittest
+
+
+class SeleniumTests(unittest.TestCase):
+    def setUp(self):
+        chrome_options = Options()
+        chrome_options.add_argument('--headless')
```

Comment:
I forgot to comment this but headless mode essentially means it doesn't open chrome like it normally would and show exactly what it's doing.